scene/gui/text_edit.cpp

```diff
@@ -3168,6 +3168,11 @@ void TextEdit::_gui_input(const Ref<InputEvent> &p_gui_input) {
 			case KEY_LEFT: {
 				if (k->get_shift()) {
 					_pre_shift_selection();
+				} else if (had_selection && !k->get_command()) {
```

Comment:
Should we check for `alt` as well?

Reply:
I'm not sure what `get_alt` is supposed to be, considering pressing Alt+Arrows for me switches between documents, and doesn't actually deselect anything, suggesting that is handled earlier. Am I missing something?

Reply:
Yeah the `ScriptEditor` handles `alt` it before it reaches the `TextEdit` node. Easiest way is to add a `TextEdit` node to a scene and it test it there.

Reply:
Hmm, I tried that, and it doesn't do anything when I hold Alt and press the arrow keys. No deselect, no moving the caret. Tested on Windows & Linux.

Reply:
That's what I would expect, though It seems to move the caret and deselect the text for me.